logzio/resource_alert_v2.go

```diff
@@ -292,9 +294,19 @@ func resourceAlertV2Create(d *schema.ResourceData, m interface{}) error {
 	alertId := strconv.FormatInt(a.AlertId, BASE_10)
 	d.SetId(alertId)
 
-	// A temporary solution to handle a 404 error that happens otherwise
-	time.Sleep(delayGet * time.Second)
-	return resourceAlertV2Read(d, m)
+	return retry.Do(
+		func() error {
+			return resourceAlertV2Read(d, m)
+		},
+		retry.RetryIf(
+			func(err error) bool {
+				if strings.Contains(err.Error(), "missing alert"){
```

Comment:
does the err msg contains "404"? if so, better to search for it and not "missing alert"

Reply:
The client returns the following message: "API call GetAlert failed with missing alert %d, data: %s"